web/src/components/scaffolding/Box.tsx

```diff
@@ -0,0 +1,9 @@
+import { styled } from "@/styled-system/jsx";
```

Comment:
```suggestion
import { styled, Box, Flex } from "@/styled-system/jsx";
```

Panda also exposes components like Box, Flex, Stack, HStack, VStack just like Chakra

Reply:
Oh amazing, I didn't know about that, thanks!